libp2p/multibase.nim

```diff
@@ -11,31 +11,34 @@
 ##
 ## TODO:
 ## 1. base32z
+## 
+
+{.push raises: [Defect].}
+
 import tables
-import stew/[base32, base58, base64]
+import stew/[base32, base58, base64, results]
+export results
```

Comment:
@arnetheduck this causes nim compiler to fail and error:
```
/Users/travis/gopath/src/github.com/status-im/nim-libp2p/libp2p/multiaddress.nim(807, 33) Error: type mismatch: got <int literal(6)> but expected 'Protocol = enum'
```
Only on macos/linux so far

Reply:
Well I'm ignoring the issue for now.. removed `export results`
I hope that is fine for now...